activation/handler.go

```diff
@@ -408,14 +412,14 @@ func (h *Handler) storeAtx(ctx context.Context, atx *types.VerifiedActivationTx)
 	}
 	if proof != nil {
 		h.cdb.CacheMalfeasanceProof(atx.SmesherID, proof)
+		h.tortoise.OnMalfeasance(atx.SmesherID)
```

Comment:
i just realized that tortoise.OnMalfeasance() will be called twice. once here, and once when Publish() is called on the malfeasance proof https://github.com/spacemeshos/go-spacemesh/pull/4555/files#diff-32a06abd05b71236130e135e1db758587402f061b1fa6dd766248333fafcc293R442

Reply:
it can be called any number of times, only the first one matters